pkgs/data/sgml+xml/schemas/xml-dtd/docbook/generic.nix

```diff
@@ -3,8 +3,8 @@
 stdenv.mkDerivation {
   inherit src name postInstall;
 
-  nativeBuildInputs = [unzip];
-  propagatedNativeBuildInputs = [ findXMLCatalogs ];
+  nativeBuildInputs = [ unzip ];
+  propagatedBuildInputs = [ findXMLCatalogs ];
```

Comment:
This is a actually a bug I think. "propagatedNativeBuildInputs" should be correct. Will have a fix soon hopefully.

Reply:
I guess I can wait a bit, I just get the things working properly and not blocking on nitpicks.